reference/pickup.v3.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.1
+info:
+  title: Pickup
+  version: '3.0'
+  termsOfService: 'https://www.bigcommerce.com/terms'
+servers:
+  - url: 'https://api.bigcommerce.com'
+    variables:
+      store_hash:
+        default: store_hash
+paths:
+  '/stores/{store_hash}/v3/pickup/methods':
+    delete:
+      tags:
+        - Pickup Methods
+      description: Delete existing pickup methods.
+      parameters:
+        - name: id:in
```

Comment:
id?

Reply:
It actually is `id:in`, since it's a comma separated list of IDs, so we're saying find those with ID _in_ this list, for example [in customers API](https://github.com/bigcommerce/api-specs/blob/bdf5c3e41746c2636065c3bf17e18759ef5834c7/reference/customers.v3.yml#L90).